install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -347,6 +354,47 @@ storage that are registered with it. Trident itself is configured separately.
 <1> For more information about installing Trident with {product-title}, see the link:https://github.com/NetApp/trident[Trident documentation].
 <2> For more information about supported parameters, see the link:https://github.com/NetApp/trident#storage-attributes[storage attributes] section of the Trident documentation.
 
+[[change-default-storage-class]]
+== Changing the default StorageClass
+Use the following process to change the default StorageClass:
+
+. List the StorageClass:
++
+====
+----
+$ oc get storageclass
+
+NAME                 TYPE
+gp2 (default)        kubernetes.io/aws-ebs <1>
+standard             kubernetes.io/gce-pd
+----
+<1> `(default)` denotes the default StorageClass.
+====
+
+. Change the value of the annotation `storageclass.kubernetes.io/is-default-class` to `false` for the default StorageClass:
++
+----
+$ oc patch storageclass gp2 -p '{"metadata": {"annotations": \
+{"storageclass.kubernetes.io/is-default-class": "false"}}}'
```

Comment:
To be consistent with the rest of the docs, I think two tabs are needed before the second line. Here's a random example: https://docs.openshift.com/container-platform/3.5/dev_guide/managing_images.html#private-registries